MediaBrowser.Providers/MediaInfo/AudioResolver.cs

```diff
@@ -172,5 +169,48 @@ private Task<Model.MediaInfo.MediaInfo> GetMediaInfo(string path, CancellationTo
                 },
                 cancellationToken);
         }
+
+        private MediaStream DetectLanguage(MediaStream mediaStream, string fileNameWithoutExtension, string videoFileNameWithoutExtension)
```

Comment:
Looks duplicated between resolvers, if there are no differences I'd create a parent class to pull this up to.